bolt/util.py

```diff
@@ -94,8 +110,186 @@ def get_qualified_vcf_annotation(anno_enum):
     assert anno_enum in constants.VcfInfo or anno_enum in constants.VcfFormat
     return f'{anno_enum.namespace}/{anno_enum.value}'
 
+def split_vcf(input_vcf, output_dir):
```

Comment:
Since the below code is specific to PCGR, it would be best to move it to the `pcgr.py` file to avoid having single-use code in `util.py`